src/dos/cdrom_ioctl_linux.cpp

```diff
@@ -229,6 +229,11 @@ bool CDROM_Interface_Ioctl::GetMediaTrayStatus(bool& mediaPresent,
 	return true;
 }
 
+bool ReadSector(uint8_t* buffer, const bool raw, const uint32_t sector) override
```

Comment:
As per our discussion in my recent PR, feel free to start migrating things toward simple `int`s (unless it would be a bigger refactoring job and you're not ready for that right now, which is fine 😄 ).

Reply:
I have a cleanup PR in the works (that's how I found out about the pointer ownership), so I'll add this to that PR.